### DISCUSSION TITLE
Possible ways to harden NetBox

### DISCUSSION THREAD

**Discussion by imix99** [netbox-community/netbox#16340] [Help Wanted!]
Hello everyone,

I have just completed my installation and implementation of NetBox and would now like to harden NetBox. 
Do you have any suggestions or checklists for hardening NetBox

Thanks in advance

**Comment by mr1716** [comment]
Check out these built In setting [Security Related Settings](https://demo.netbox.dev/static/docs/configuration/optional-settings/)

Make sure to set items like Login Timeout according to best practices for your org and other settings as needed. 

This is a great start.
